docs/bc-registries-audit-notification-workflow.md

```diff
@@ -88,4 +88,12 @@ Specifics regarding the commands referenced in this workflow can be found here:
 
 ## Audit scripts for Aries VCR/OrgBook and BC Registries Issuer
 
-This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
\ No newline at end of file
+This repository provides scripts to audit the OrgBook search database and agent wallet against the source BC Registries data; [Audit Scripts](https://github.com/bcgov/von-bc-registries-audit/blob/main/README.md#understanding-the-output)
+
+## Troubleshoot Failed Credentials Error
+
+Occasionally credentials get stuck in the processing queue due to errors, and will not respond to the use of the [`requeueOrganization`](./bc-registries-agent-configurations-manage-script.md#requeueOrganization-command) and/or [`queueOrganization`](./bc-registries-agent-configurations-manage-script.md#queueOrganization-command) commands.  In these cases you want to check the pipeline status for an errors.
+
+Running the [`getPipelineStatus` command](./bc-registries-agent-configurations-manage-script.md#requeueOrganization-command) command will verify the credentials errors.
+
+After verifying the credentials have been requeued, run the [`runPipeline` ](./bc-registries-agent-configurations-manage-script.md#requeueOrganization-command) command to process the outstanding records.
```

Comment:
You removed too much content from this section and did not add the other recommendations I made.  Also the screenshot from `getPipelineStatus` got removed from the PR.

I see there may have been some confusion regarding this comment; https://github.com/bcgov/DITP-DevOps/pull/54#discussion_r1114383899

Is was referencing this paragraph only, all other content should have stayed:

> Running the [`requeueOrganization` command](./bc-registries-agent-configurations-manage-script.md#requeueOrganization-command) and [`queueOrganization` command](./bc-registries-agent-configurations-manage-script.md#queueOrganization-command) will resolve the data issue.

Reply:
Hi @WadeBarnes made the changes according to your suggestion